[BUG]Stripe:3Dセキュア有効なカードでの決済時にシステムエラー

## 対象
:sob: その他

## バグの説明
・3Dセキュアが有効なカードで決済した際にシステムエラーが発生する。

**システムエラーの内容**

```
[2020-02-17 11:11:02 +0000] [339] [INFO] payment.Stripe.Charge
INFO:stripe:message='Request to Stripe api' method=post path=https://api.stripe.com/v1/charges
INFO:stripe:message='Stripe API response' path=https://api.stripe.com/v1/charges response_code=402
INFO:stripe:error_code=authentication_required error_message='Your card was declined. This transaction requires authentication.' error_param=None error_type=card_error message='Stripe API error received'
[2020-02-17 11:11:03 +0000] [339] [ERROR] Error handling request /v2/Stripe/Charge
Traceback (most recent call last):
File "/home/apl/.pyenv/versions/3.6.8/lib/python3.6/site-packages/gunicorn/workers/base_async.py", line 55, in handle
self.handle_request(listener_name, req, client, addr)
File "/home/apl/.pyenv/versions/3.6.8/lib/python3.6/site-packages/gunicorn/workers/ggevent.py", line 143, in handle_request
super().handle_request(listener_name, req, sock, addr)
File "/home/apl/.pyenv/versions/3.6.8/lib/python3.6/site-packages/gunicorn/workers/base_async.py", line 106, in handle_request
respiter = self.wsgi(environ, resp.start_response)
File "/home/apl/.pyenv/versions/3.6.8/lib/python3.6/site-packages/falcon/api.py", line 244, in __call__
responder(req, resp, **params)
File "/home/apl/.pyenv/versions/3.6.8/lib/python3.6/site-packages/falcon/hooks.py", line 221, in do_before
responder(self, req, resp, **kwargs)
File "/app/ibet-Wallet-API/app/api/payment/stripe.py", line 533, in on_post
stripe_charge.status = StripeChargeStatus.ERROR.value
File "/home/apl/.pyenv/versions/3.6.8/lib/python3.6/enum.py", line 326, in __getattr__
raise AttributeError(name) from None
AttributeError: ERROR
```

**原因**
Enumの値誤り。

--- a/app/api/payment/stripe.py
+++ b/app/api/payment/stripe.py
@@ -481,7 +481,7 @@ class Charge(BaseResource):
         if stripe_charge is not None:  # 既にオペレーションを1回以上実施している場合
             # 二重課金のチェック
             #  - Charge状態が[PENDING]、[SUCCESS]の場合はエラー
-            #  - Charge状態が[ERROR]の場合は[PENDING]に更新（この時点でcommitする）
+            #  - Charge状態が[FAILED]の場合は[PENDING]に更新（この時点でcommitする）
             if stripe_charge.status == StripeChargeStatus.PENDING.value or \
                     stripe_charge.status == StripeChargeStatus.SUCCEEDED.value:
                 description = "Double charge"
@@ -530,45 +530,45 @@ class Charge(BaseResource):
                 description=description_agreement
             )
         except stripe.error.CardError as e:
-            stripe_charge.status = StripeChargeStatus.ERROR.value
+            stripe_charge.status = StripeChargeStatus.FAILED.value
             body = e.json_body
             err = body.get('error', {})
             raise StripeErrorClient(code=50, description='[stripe]card error caused by %s' % err.get('param'),
                                     title=err.get('message'))
         except stripe.error.RateLimitError as e:
-            stripe_charge.status = StripeChargeStatus.ERROR.value
+            stripe_charge.status = StripeChargeStatus.FAILED.value
             body = e.json_body
             err = body.get('error', {})
             raise StripeErrorClient(code=51, description='[stripe]rate limit error', title=err.get('message'))
         except stripe.error.InvalidRequestError as e:
-            stripe_charge.status = StripeChargeStatus.ERROR.value
+            stripe_charge.status = StripeChargeStatus.FAILED.value
             body = e.json_body
             err = body.get('error', {})
             raise StripeErrorClient(code=52, description='[stripe]invalid request error caused by %s' % err.get('param'),
                                     title=err.get('message'))
         except stripe.error.AuthenticationError as e:
-            stripe_charge.status = StripeChargeStatus.ERROR.value
+            stripe_charge.status = StripeChargeStatus.FAILED.value
             body = e.json_body
             err = body.get('error', {})
             raise StripeErrorServer(code=53, description='[stripe]authentication error', title=err.get('message'))
         except stripe.error.APIConnectionError as e:
-            stripe_charge.status = StripeChargeStatus.ERROR.value
+            stripe_charge.status = StripeChargeStatus.FAILED.value
             body = e.json_body
             err = body.get('error', {})
             raise StripeErrorServer(code=54, description='[stripe]api connection error', title=err.get('message'))
         except stripe.error.StripeError as e:
-            stripe_charge.status = StripeChargeStatus.ERROR.value
+            stripe_charge.status = StripeChargeStatus.FAILED.value
             body = e.json_body
             err = body.get('error', {})
             raise StripeErrorServer(code=55, description='[stripe]stripe error', title=err.get('message'))
         except Exception as err:
-            # Charge状態を[ERROR]ステータスに更新する
+            # Charge状態を[FAILED]ステータスに更新する
             stripe_charge.status = StripeChargeStatus.FAILED.value
             LOG.error('Error: %s', err)
             raise AppError
 
         if charge.status != 'succeeded':
-            # Charge状態を[ERROR]ステータスに更新する
+            # Charge状態を[FAILED]ステータスに更新する
             stripe_charge.status = StripeChargeStatus.FAILED.value
             raise InvalidCardError
 

